src/AlertBar/Dismiss.js

```diff
@@ -1,22 +1,21 @@
 import React from 'react'
+import cx from 'classnames'
 import propTypes from '@dhis2/prop-types'
 import { spacers } from '../theme.js'
 import { Close } from '../icons/Close.js'
+import { iconStyles } from './styles.js'
 
 const Dismiss = ({ onClick }) => (
     <div onClick={onClick}>
-        <Close />
+        <Close className={cx(iconStyles.className, 'dismiss')} />
```

Comment:
Aren't these in the wrong order? I expect `cx('dismiss', iconStyles.className)`.

Does that change anything?

Reply:
I suppose that would be more readable, so I'll go ahead and change that.... However it doesn't change anything.